libraries/AP_Mount/AP_Mount_Xacti.cpp

```diff
@@ -560,11 +568,11 @@ bool AP_Mount_Xacti::handle_param_get_set_response_string(AP_DroneCAN* ap_dronec
             _firmware_version.mav_ver = UINT32_VALUE(dev_ver_num, patch_ver_num, minor_ver_num, major_ver_num);
         }
         return false;
-    } else if (strcmp(name, XACTI_PARAM_DATETIME) == 0) {
+    } else if (strcmp(name, get_param_name_str(Param::DateTime)) == 0) {
         // display when time and date have been set
         gcs().send_text(MAV_SEVERITY_INFO, "%s datetime set %s", send_text_prefix, (const char*)value.data);
         return false;
-    } else if (strcmp(name, XACTI_PARAM_STATUS) == 0) {
+    } else if (strcmp(name, get_param_name_str(Param::Status)) == 0) {
```

Comment:
```suggestion
    } else if (streq(name, get_param_name_str(Param::Status))) {
```

... this won't work, but it should; I define this in various places...

Reply:
ok thanks.  I might leave this change out until streq is defined somewhere more central.